### ISSUE TITLE
Improve Theia server start time in Che 7 workspaces.

### ISSUE THREAD

**Issue by amisevsk** [redhat-developer/rh-che#1081]
When starting a Che 7 workspace, there is a phase at the end where Che waits for all servers to be started. The Theia server consistently takes 10 seconds to start, regardless of anything else, as marked in blue below:

![chart 3](https://user-images.githubusercontent.com/16168279/48725363-f5eaf400-ebf9-11e8-9eee-4c5670e87693.png)

It appears that a large part of this wait is due to the way [ServersChecker](https://github.com/eclipse/che/blob/9cebfa1ffcf37dabfeac7a57de7d62ca44220d72/wsmaster/che-core-api-workspace/src/main/java/org/eclipse/che/api/workspace/server/hc/ServersChecker.java#L226) is implemented, where there is a 3 second delay between server liveness checks. 

Additionally, the configmap we use for rhche [requires two successful pings](https://github.com/redhat-developer/rh-che/blob/af1f34dd80a777b4d3f744d6ac22ad6c60e0d35e/openshift/rh-che.config.yaml#L44) before a server is marked as started. Combined with above, this means that the minimum amount of time for a server to start is 6 seconds, and often more if the first ping fails.

We should:
- [x] Add functionality upstream to control the ping interval.
- [x] Investigate if it is possible to decrease the ping success threshold back to the default value of 1 or if that causes issues.
- [x] Find a reasonable value for ping interval on OpenShift clusters and apply it.  

Note: there are plans to move away from the current implementation and instead use built-in liveness probes on k8s/OpenShift, but for now solving this issue could speed up start time significantly.

**Comment by amisevsk** [comment]
PR: for the property: https://github.com/eclipse/che/pull/11984

**Comment by amisevsk** [comment]
I've tested starting workspaces on the dev-cluster (using the prod-preview cluster for workspaces) and basically every increase to ping success threshold and interval increases workspace start by that much time on average (e.g. going from 500ms to 1s intervals adds ~500ms to start time, requiring two successes adds another second). 

I think our best bet at the moment is to start at a low value (e.g. 500ms and 1 success) and increase from there if we find it causes issues.

Also, I spent a while testing Che6 and 7 workspaces with a 1 ping success threshold, and found no issues with route readiness (the workspace starts and works normally), so it seems the old issue is gone.
